docs/docs/installation.md

```diff
@@ -20,7 +20,9 @@ If you're just interested in playing around with React, you can use CodePen. Try
 
 If you prefer to use your own text editor, you can also <a href="/react/downloads/single-file-example.html" download="hello.html">download this HTML file</a>, edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
 
-## Creating a Single Page Application
+Currently there are two popular ways to get started with React:
+
+## 1. Using Create React App
```

Comment:
I feel like numbering it makes it seem like these are two steps to `Creating a Single Page Application`, which makes it less clear what these instructions are saying 🤔